config/adminlte.php

```diff
@@ -110,15 +122,17 @@
         [
             'text' => 'Change Password',
             'url' => 'admin/settings',
-            'icon' => 'lock'
+            'icon' => 'lock',
+            //'permission'=>'no-access',
```

Comment:
Also, I prefer `can` instead of `permission` to align with Laravel (Blade directive, user model method and middleware).